lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -62,8 +62,17 @@ void CreateSiFiveMetadataPass::renameMemory(CircuitOp circuitOp) {
   std::map<FirMemory, StringAttr> memNameMap;
   auto *ctxt = circuitOp.getContext();
   auto modNameAttr = StringAttr::get(ctxt, "modName");
-  for (auto mod : circuitOp.getOps<FModuleOp>())
+  // This is a random number to start the groupIDs at, large enough to not
+  // conflict with existing IDs.
+  // TODO: Move this logic out of this pass.
+  unsigned baseGroupID = -1;
```

Comment:
`std::numeric_limits<unsigned>::max()` might be a bit better?

Reply:
Thanks for pointing it out, replaced the -1.